packages/mobile/src/web3/actions.ts

```diff
@@ -15,7 +15,6 @@ export enum Actions {
   SET_BLOCK_NUMBER = 'WEB3/SET_BLOCK_NUMBER',
   REQUEST_SYNC_PROGRESS = 'WEB3/REQUEST_SYNC_PROGRESS',
   UPDATE_WEB3_SYNC_PROGRESS = 'WEB3/UPDATE_WEB3_SYNC_PROGRESS',
-  SET_GAS_PRICE = 'WEB3/SET_GAS_PRICE',
```

Comment:
Gas price from the store was not used. Instead, we should get it from the `getGasPrice` util which will ensure it's not stale.